command/cp.go

```diff
@@ -528,7 +528,9 @@ func (c Copy) doDownload(ctx context.Context, srcurl *url.URL, dsturl *url.URL)
 		return err
 	}
 
-	file, err := dstClient.Create(dsturl.Absolute())
+	var tempurl = *dsturl
+	tempurl.Path = tempurl.Path + ".tmp"
+	file, err := dstClient.Create(tempurl.Absolute())
```

Comment:
if the file (`$file.tmp`) exists (somehow, probably won't happen), the command will fail. Could you please try to use `ioutil.Tempfile()` which also handles file name collisions?

Reply:
+1, this would also keep users's cwd clean.